lib/analysis_server.dart

```diff
@@ -126,6 +126,11 @@ class AnalysisServer implements Disposable {
     });
 
     onReceive.listen((String message) {
+      if (message.startsWith('Observatory listening') ||
+          message.startsWith('Observatory no longer listening')) {
+        atom.notifications.addInfo('Analysis server', detail: message.trim(), dismissable: true);
```

Comment:
This message will contain the port to connect to for the observatory connection, so we'll want it to persist for a while.